### ISSUE TITLE
? How to use mixins

### ISSUE THREAD

**Issue by JesperWe** [huttonr/bootstrap3#11]
Hello! I am trying to figure how to use mixins using this package, but your documentation is slightly terse.
I have 

```
    "customVariables": true,
    "exposeMixins": true,
```

...but if I try to 

```
@import "bootstrap-mixins.less";
```

in `bootstrap-variables.less` meteor (1.2.1) hangs on startup. 
Is this the way it is supposed to work?

**Comment by huttonr** [comment]
It may be hanging as you are trying to import the bootstrap mixins from the variables file.

What's your use case for this?  Generally any less file needing to use the mixins would separately import each of these files.  Such as:

``` less
// my-custom.less
@import "bootstrap-variables.less";
@import "bootstrap-mixins.less";

/* Custom less using mixins and variables here */
```

I will later investigate this further to see why it hangs but it should work as above.

**Comment by clarkgrg** [comment]
Nice work with this.  I got it to work as stated above.  I'm new to less so your example above was very helpful.  It took me a while to figure out what to do before I came across the note above.  I may suggest you add this example to your README notes to clarify how modifications are supposed to be incorporated.  Thanks again, I spend about 3 hours trying to get the twbs:bootstrap to do what I wanted and no success.  I removed it and added your work and it worked like a champ.
